Makefile

```diff
@@ -18,6 +18,12 @@ CFLAGS ?= -O3 -I$(LVGL_DIR)/ $(WARNINGS) `pkg-config --cflags wayland-client` `p
 LDFLAGS ?= -lSDL2 -lm `pkg-config --libs wayland-client` `pkg-config --libs xkbcommon` -lpthread
 BIN = demo
 
+CONFIG_LV_USE_THORVG_INTERNAL ?=y
```

Comment:
I wouldn't add it as we can't assume ThorVg is installed.

Reply:
> I wouldn't add it as we can't assume ThorVg is installed.
This is because I added the option to compile thorvg in lvgl/lvgl.mk
```
ifeq ($(CONFIG_LV_USE_THORVG_INTERNAL),y)
   CXXSRCS += $(shell find $(LVGL_PATH)/src/libs -type f -name '*.cpp')
endif
```
Or how do you think it should be changed to be more suitable ?

Reply:
If we add the the `#if LV_USE_THORVG_INTERNAL` to all ThorVg src (as suggested [here](https://github.com/lvgl/lvgl/pull/4691#discussion_r1375471514)) no special treatment is required. If it's enabled in `lv_conf.h` it will be compiled with the other LVGL files. 

So we should handle only `LV_USE_THORVG_EXTERNAL` in Makefile and CMakeList.txt.